engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/K8sDeploymentManager.scala

```diff
@@ -33,50 +53,162 @@ class K8sDeploymentManagerProvider extends DeploymentManagerProvider {
   override def name: String = "lite-streaming-k8s"
 }
 
-class K8sDeploymentManager(modelData: ModelData, dockerImageName: String, dockerImageTag: String)
+class K8sDeploymentManager(modelData: ModelData,
+                           dockerImageName: String,
+                           dockerImageTag: String,
+                           configOverrides: Config)
                           (implicit ec: ExecutionContext, actorSystem: ActorSystem) extends BaseDeploymentManager with LazyLogging {
 
-  import skuber.json.format._
-
   private val k8s = k8sInit
 
+  private def wrapInModelConfig(config: Config): Config = {
+    ConfigFactory.parseMap(Collections.singletonMap("modelConfig", config.root()))
+  }
+
+  private val serializedModelConfig = {
+    val inputConfig = modelData.inputConfigDuringExecution
+    inputConfig.copy(config = wrapInModelConfig(inputConfig.config.withoutPath("classPath"))).serialized
+  }
+
+  private val serializedConfigOverrides = wrapInModelConfig(configOverrides).root().render(ConfigRenderOptions.concise())
+
   override def deploy(processVersion: ProcessVersion, deploymentData: DeploymentData,
                       processDeploymentData: ProcessDeploymentData,
                       savepointPath: Option[String]): Future[Option[ExternalDeploymentId]] = {
     logger.debug(s"Deploying using docker image: $dockerImageName:$dockerImageTag")
-    // TODO: implement
-    k8s.create[Pod](Pod("runtime", Pod.Spec(containers = List(Container("runtime", s"$dockerImageName:$dockerImageTag"))))).map { createResult =>
-      logger.info(s"Created pod: $createResult")
+    val objectName = objectNameForVersion(processVersion)
+    val scenario = processDeploymentData.asInstanceOf[GraphProcess].processAsJson
+
+    val labels = Map(
+      scenarioNameLabel -> sanitizeNameLabel(processVersion.processName),
+      scenarioIdLabel -> processVersion.processId.value.toString,
+      scenarioVersionLabel -> processVersion.versionId.value.toString
+    )
+
+    Future.sequence(List(
+      k8s.create[ConfigMap](ConfigMap(
+        metadata = ObjectMeta(
+          name = objectName,
+          labels = labels
+        ), data = Map(
+          "scenario.json" -> scenario,
+          "modelConfig.conf" -> serializedModelConfig,
+          "configOverrides.conf" -> serializedConfigOverrides
```

Comment:
Why we have separate `modelConfig` and `configOverrides` instead of just `modelConfigAfterOverride`?

Reply:
hmm... guess it was easier for me to debug... ;) changed it